app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({self.name: self})
```

Comment:
In the `__init__` method, the `people` dictionary should be accessed using `Person.people` instead of `self.people`. This ensures that the class attribute is updated correctly. Change `self.people.update({self.name: self})` to `Person.people.update({self.name: self})`.